### ISSUE TITLE
OSError: [WinError -2147483392] Windows Error 0x80000100

### ISSUE THREAD

**Issue by Dosclic98** [pastapojken/Fusion360WineInstaller#6]
Hi, when the installation finishes it gives me this error.
[autodesk.webdeploy.streamer.log](https://github.com/pastapojken/Fusion360WineInstaller/files/3588010/autodesk.webdeploy.streamer.log)
And also during the installation it says that one of the programs crashes but i will give you further infos when i test it again. Using wine version wine-3.0.1 (Debian 3.0.1-2). This is the only one that let me see the status bar while installing (with stable, dev and staging the bar doesn't show up).
EDIT
The error that occurs during the installation regards the Material Library migrator but i think that this is a minor problem because i read that after the complete installation that executable can be ran to fix the problem using a workaround.
[backtrace.txt](https://github.com/pastapojken/Fusion360WineInstaller/files/3588068/backtrace.txt)
If i try to start Fusion360Launcher.exe it says that FusionLauncher.exe.ini might be missing or incomplete and says to reinstall Fusion.

**Comment by farfalleflickan** [comment]
Hi! Have you tried using a newer version of wine?

**Comment by Dosclic98** [comment]
Yes, if I use the 4.0 version things get worse and I don't even get the installation bar, it gets stuck

**Comment by JacekJagosz** [comment]
I had problems similar problems with older Wines, but with 4.15 it works flawlessly. Any chance you can get a more current version, all versions you listed are very old.

**Comment by Dosclic98** [comment]
Installed same version as yours but doesn't work, bar don't appear and even if it creates this three files in with the executable It doesn't work. Did you use some particular settings? How did you install it?

**Comment by JacekJagosz** [comment]
The thing is, during my installation there wasn't at any point a progress bar.
Mine went like this:

1. I run `installer.sh 1`, winecfg, vcrun, and finally fusion window appears. After a while (after files get downloaded) it disappears for a few seconds, pops up again and then login window appears. 
2. After log in, it disappears and you are left with only 1 window. After a while you need to kill it.
3. I run `installer.sh 2` then `installer.sh 3` and after 30s I get Fusion splash screen and then actual program window. There you are supposed to change graphics driver from "autodetect" to "directx 9" and quit.
4. After running `installer.sh 2` again I can simply launch Fusion through application menu and everything works.
At which point does it fail for you?

**Comment by Dosclic98** [comment]
The login window don't even appear, also continuing with further steps don't work. I think that if you don't get the login it gets stuck. I am trying now to redo the entire process another time then I will try other versions of wine
Do I have to set wine to 32bit enabling i836 architecture?

**Comment by Dosclic98** [comment]
Ok guys i'm back here to say that it's working!! When i first tried again with wine developement 4.15 i managed to arrive to the login window and after an error i tried to start the program and it gave me an unhandled exception error even installing DXVK before or not. 
After that i tryed with wine 4.15 staging and i still arrived to the login window but without having the same error of before, so i installed DXVK and started fusion but i got the same unhandled exception as the previous time. After that i uninstalled DXVK, i started fusion and everything is working: design, model studyes, toolbar, cloud saving, cloud exporting even without setting it to DX9.
What should not work on DX11 that works on DX9? Are there some majour problems that i am not experiencing now using it with DX11?
In conclusion of this issue i would definitely suggest to install a newer version of wine and try different branches. Also one of my problems even with the first test on newer wine version was that i probably killed the program too early when i thought it was stuck but i should have let it go, so at least you should give him 15/30 minutes to be sure the program is stuck. One last thing is to test everything with and without DXVK because in my case it broke everything and gave only problems.
